### DISCUSSION TITLE
Photo editor similar to google photos/one

### DISCUSSION THREAD

**Discussion by jagjordi** [immich-app/immich#3122] [General]
@alextran1502  has stated several times that the main goal of immich is to imitate google photos as close as possible. Now its comming to a point where the main functionalities of photo backup and sharing/organization are already almost equivalent to google photos (in some cases even better).

However one major thing I still need to use google photos for is to edit pictures, mainly do some color correction, enhancement cropping, rotating, etc.

Is there any plans to integrate those features in immich at some point?

On a side note, does anyone know about some open source tool that can do the type of edit/adjustments that google photos can? I'm manily interested in the automatic color "enhancement".

**Comment by HeyVern** [comment]
FYI: Nextcloud is opensource...and they have a done a pretty good job of basic photo editing.  Maybe there is some code that might be reusable.  (have not looked at code yet, just have used the nextcloud app)

**Comment by linuxturtle** [comment]
I will add that basic crop/rotate functionality is a spousal unit hard requirement for me in any attempt at self-hosting a photo repository.  No way can immich (or any other self-hosted photo app I've tried) replace google photos for my family without it.  In searching, I see that https://github.com/immich-app/immich/issues/768 was closed by @alextran1502 a year ago as "Out of scope", but maybe he'd reconsider at this point?

**Reply by alextran1502** [reply]
@linuxturtle hey, there is an on going PR and discussion for editing photo by @faupau03 , do you guys want to discuss the colab on this feature?

**Reply by linuxturtle** [reply]
@alextran1502 sure!  Where is the discussion taking place?

**Reply by linuxturtle** [reply]
NM, I think I found https://github.com/immich-app/immich/pull/3271 -- is that it?

**Reply by alextran1502** [reply]
@linuxturtle yes and on Discord in Focus-discussion

**Reply by linuxturtle** [reply]
OK, thx.  I don't grok discord.  I clicked on the link in the PR, but just get something about not having permission for text discussions :/

**Reply by linuxturtle** [reply]
NM again, I think I figured it out.  I wasn't on the immich discord server yet.

**Comment by kwongtn** [comment]
Just came across photopea, probably can be used as base editor (or even like the 'advanced' editor?
https://www.photopea.com/

**Reply by bo0tzz** [reply]
photopea isn't open source, so I'm not sure how you see that working?
